
The following have been reloaded with a version change:
  1) python3/3.7.0 => python3/3.9.2

Traceback (most recent call last):
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/train_dfsr.py", line 8, in <module>
Traceback (most recent call last):
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/train_dfsr.py", line 8, in <module>
    from utils.models import DFSRNet
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/../utils/models.py", line 7, in <module>
Traceback (most recent call last):
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/train_dfsr.py", line 8, in <module>
    from utils.models import DFSRNet
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/../utils/models.py", line 7, in <module>
Traceback (most recent call last):
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/train_dfsr.py", line 8, in <module>
    import config 
ModuleNotFoundError: No module named 'config'
    import config 
ModuleNotFoundError: No module named 'config'
    from utils.models import DFSRNet
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/../utils/models.py", line 7, in <module>
    import config 
ModuleNotFoundError: No module named 'config'
Traceback (most recent call last):
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/train_dfsr.py", line 8, in <module>
    from utils.models import DFSRNet
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/../utils/models.py", line 7, in <module>
    import config 
ModuleNotFoundError: No module named 'config'
Traceback (most recent call last):
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/train_dfsr.py", line 8, in <module>
    from utils.models import DFSRNet
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/../utils/models.py", line 7, in <module>
    from utils.models import DFSRNet
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/../utils/models.py", line 7, in <module>
    import config 
ModuleNotFoundError: No module named 'config'
Traceback (most recent call last):
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/train_dfsr.py", line 8, in <module>
    import config 
ModuleNotFoundError: No module named 'config'
    from utils.models import DFSRNet
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/../utils/models.py", line 7, in <module>
Traceback (most recent call last):
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/train_dfsr.py", line 8, in <module>
    import config 
ModuleNotFoundError: No module named 'config'
    from utils.models import DFSRNet
  File "/work2/10214/yu_yao/frontera/deep-fourier-based-super-resolution-net/src/../utils/models.py", line 7, in <module>
    import config 
ModuleNotFoundError: No module named 'config'
srun: error: c198-092: tasks 4-7: Exited with exit code 1
srun: error: c198-091: tasks 0-3: Exited with exit code 1
